components/filterTodos.js

```diff
@@ -0,0 +1,35 @@
+import { renderTodoList } from "./domActions.js";
+import { setCurrentFilterState } from "../entry.js";
+import {
+  filterButtonWrapper,
+  filterAllButton,
+  filterCompleteButton,
+  filterIncompleteButton,
+} from "./domElements.js";
+
+const filterButtons = filterButtonWrapper.getElementsByClassName("button");
```

Comment:
The class name is too generic and can cause unexpected bugs. Let's use a specific name
```suggestion
const filterButtons = filterButtonWrapper.getElementsByClassName("filter-button");
```

Reply:
It has to be generic as it contains global styles. That's why I targeted the wrapper first and then the buttons.